src/main/scala/mesosphere/marathon/core/task/tracker/impl/InstanceOpProcessorImpl.scala

```diff
@@ -19,7 +19,6 @@ import scala.util.control.NonFatal
   */
 private[tracker] class InstanceOpProcessorImpl(
```

Comment:
if we merge this, the next easy step will be to get rid of this whole class because it contains almost nothing right now. I kept it here because it will be easier to review this way...

Reply:
Do you have a ticket for follow up?

Reply:
no, I was planning to do it in the same ticket, just as a separate PR